### PR TITLE
Parent onboarding

### PR DESCRIPTION
**Pull request by DerangedMind**
Something messed up in #33, so I'm creating a new PR.

### CODE REVIEW

File: app/controllers/director/kids_controller.rb [hunk 1]
```diff
@@ -24,12 +21,17 @@ def create
...
+        redirect_to director_kid_path(:id => @kid.id)
+      else
+        @kid.delete
+        flash[:notice] = @kid_medical.errors.full_messages
+        redirect_to new_director_kid_path
+      end
     else
-      puts "KID NOT SAVED"
-      puts @kid.errors.full_messages
+      flash[:notice] = @kid.errors.full_messages
```
Comment (Garo-A) [inline app/controllers/director/kids_controller.rb]: Use `flash[:alert]`here. The error messages will show up in red.

Comment (DerangedMind) [inline app/controllers/director/kids_controller.rb]: Fixed!

**Comment by TheTempas** [comment]
:+1:

File: app/views/parent/kids/index.html.erb [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-  <% @kids.each do |kid| %>
-    <tr>
-      <td> <%= kid.first_name %> </td>
-      <td> <%= kid.last_name %> </td>
-      <td> <%= kid.birthdate %> </td>
-      <td> <%= kid.sin %> </td>
-      <td> <%= link_to "Edit Kid", edit_parent_kid_path(:id => kid.id) %> </td>
-    </tr>
-  <% end %>
-</table>
```
Comment (TheTempas) [inline app/views/parent/kids/index.html.erb]: Why did you delete the entire parents/kids/index page?

Comment (DerangedMind) [inline app/views/parent/kids/index.html.erb]: Please see https://github.com/DerangedMind/fluffy-camp-fire/pull/33

We decided to scrap it and just show all the info on the profile page.
